barretenberg/ts/src/barretenberg_api/pedersen.test.ts

```diff
@@ -28,6 +28,34 @@ describe('pedersen', () => {
     expect(result).toEqual(new Fr(2152386650411553803409271316104075950536496387580531018130718456431861859990n));
   });
 
+  it('pedersenCompressAndHashSame', async () => {
```

Comment:
These are a bunch of "crutch" tests that show that a lot of the pedersen methods are effectively the same or some are just supersets.

Once I remove the c_binds for them, these will also disappear since they won't be available anymore